spec/lib/cli_spec.rb

```diff
@@ -161,13 +161,29 @@
     end
   end
 
-  describe ':compile option' do
-    let(:options)  { {:compile => true, :evals => ['puts 2342']} }
+  describe ':runner option' do
+    context 'when :compile' do
+      let(:options)  { {runner: :compiler, evals: ['puts 2342']} }
 
-    it 'outputs the compiled javascript' do
-      expect_output_of{ subject.run }.to include(".$puts(2342)")
-      expect_output_of{ subject.run }.not_to include("2342\n")
+      it 'outputs the compiled javascript' do
+        expect_output_of{ subject.run }.to include(".$puts(2342)")
+        expect_output_of{ subject.run }.not_to include("2342\n")
+      end
+
+      context 'with the :map_file runner option' do
+        let(:map_file) { "#{Dir.mktmpdir 'opal-map'}/file.map" }
+        let(:runner_options)  { {map_file: map_file, evals: ['123']} }
```

Comment:
Is `:eval` field necessary here? From what I understand the builder doesn't use it

Reply:
@iliabylich good catch, fixed